paasta_tools/cleanup_marathon_jobs.py

```diff
@@ -118,45 +134,68 @@ def delete_app(app_id, client, soa_dir):
         raise
 
 
-def cleanup_apps(soa_dir):
+def cleanup_apps(soa_dir, kill_threshold=0.5, force=False):
     """Clean up old or invalid jobs/apps from marathon. Retrieves
     both a list of apps currently in marathon and a list of valid
     app ids in order to determine what to kill.
 
-    :param soa_dir: The SOA config directory to read from"""
+    :param soa_dir: The SOA config directory to read from
+    :param kill_threshold: The decimal fraction of apps we think is
+        sane to kill when this job runs.
+    :param force: Force the cleanup if we are above the kill_threshold"""
     log.info("Loading marathon configuration")
     marathon_config = marathon_tools.load_marathon_config()
     log.info("Connecting to marathon")
     client = marathon_tools.get_marathon_client(marathon_config.get_url(), marathon_config.get_username(),
                                                 marathon_config.get_password())
 
-    valid_services = get_services_for_cluster(instance_type='marathon', soa_dir=soa_dir)
+    valid_apps = get_services_for_cluster(instance_type='marathon', soa_dir=soa_dir)
```

Comment:
can you make this 'valid_services' again here please - it makes it easier to understand that this is the services declared in soa-configs vs the 'apps' that are running in marathon